client/client_root_validation_test.go

```diff
@@ -34,45 +33,12 @@ func validateRootSuccessfully(t *testing.T, rootType string) {
 	err := repo.tufRepo.InitTimestamp()
 	require.NoError(t, err, "error creating repository: %s", err)
 
-	// Initialize is supposed to have created new certificate for this repository
-	// Lets check for it and store it for later use
-	allCerts := repo.CertStore.GetCertificates()
-	require.Len(t, allCerts, 1)
-
 	fakeServerData(t, repo, mux, keys)
 
 	//
-	// Test TOFUS logic. We remove all certs and expect a new one to be added after ListTargets
+	// Test TOFUS logic.
```

Comment:
Would it make sense to just delete this file?  I think we already have tests for listing targets on an empty repo in `client_tests.go`, although maybe not with an RSA key for root.